src/main/kotlin/util/DbUtil.kt

```diff
@@ -50,7 +61,7 @@ fun Transaction.putBinary(id: String, value: ByteArray) {
 
 fun Transaction.getBinary(id: String): ByteArray {
     val prepareStatement = connection.prepareStatement(
-        "SELECT BINARYA FROM BINARYA WHERE id = '$id'",
+        "SELECT BINARYA FROM BINARYA WHERE $ID_COLUMN = '$id'",
```

Comment:
```suggestion
        "SELECT BINARYA FROM BINARYA WHERE $ID_COLUMN = ${id.toQuotes()}",
```
Why we are not using toQuotes everywhere ?